aidants_connect_habilitation/models.py

```diff
@@ -332,6 +340,7 @@ def draft_id(self):
     class Meta:
         verbose_name = "aidant à habiliter"
         verbose_name_plural = "aidants à habiliter"
+        unique_together = (("email", "organisation"),)
```

Comment:
Pour les demandes d'habilitation côté "web" j'avais humanisé un peu le message : 

```suggestion
        unique_together = (("email", "organisation"),)
        error_messages = {
            NON_FIELD_ERRORS: {
                "unique_together": (
                    "Il y a déjà un aidant avec la même adresse e-mail dans "
                    "cette organisation. Chaque aidant doit avoir son propre "
                    "e-mail nominatif."
                ),
            }
        }
```

Reply:
(…je crois que c'est là que ça se met, faudra ptêt vérifier que çaMarche™ quand même)

Reply:
C'est dommage qu'on puisse pas personnaliser le message `unique_together` en fonction des champs concernés. Bon, ici je doute qu'on ai un autre contrainte `unique_together` mais si c'était le cas, ça rendrait le message vraiment incompréhensible.

Reply:
Je crois me souvenir que le message non personnalisé mentionne les champs (genre "il existe déjà un objet truc pour ce machin"). Mais en effet je ne crois pas que l'on puisse individualiser le message personnalisé par champ.